pkg/phases/audit/test.go

```diff
@@ -0,0 +1,144 @@
+package audit
+
+import (
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/flanksource/commons/console"
+	"github.com/moshloop/platform-cli/pkg/platform"
+	v1 "k8s.io/api/core/v1"
+)
+
+const jUnitAuditConfigClass = "AuditConfig"
+const jUnitAuditAPIServerStateClass = "AuditApiServerState"
+const jUnitAuditLogs = "AuditLogsPresent"
```

Comment:
We can probably just group everything under a single test name
```suggestion
const testName = "auditing"
```

Reply:
done